### PR TITLE
finished authorization with pundit

### PR DESCRIPTION
**Pull request by RJDPI**

### CODE REVIEW

**Review by Samuel-Lubliner — CHANGES_REQUESTED** [review]
When I sign in as Alice and try to view the user profile `/alice` I get the error Sprockets::Rails::Helper::AssetNotFound at /alice
The asset "." is not present in the asset pipeline.

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -77,17 +86,17 @@ def comment_params
     params.require(:comment).permit(:author_id, :photo_id, :body)
   end
 
-  def ensure_current_user_is_author
-    if current_user != @comment.author
-      redirect_back fallback_location: root_url, alert: "You are not authorized to do that."
-    end
-  end
+  # def ensure_current_user_is_author
```
Comment (heratyian) [inline app/controllers/comments_controller.rb:89]: You can delete this commented out code

File: app/controllers/follow_requests_controller.rb [hunk 2]
```diff
@@ -2,27 +2,28 @@ class FollowRequestsController < ApplicationController
   before_action :set_follow_request, only: %i[ show edit update destroy ]
 
   # GET /follow_requests or /follow_requests.json
-  def index
-    @follow_requests = FollowRequest.all
-  end
+  # def index
```
Comment (heratyian) [inline app/controllers/follow_requests_controller.rb:5]: same here. better to just delete

**Review by heratyian — COMMENTED** [review]
Remember to delete commented out code. Can always recover later in git commit history.

**Pull request closed without merging**